packages/tracing/src/browser/metrics.ts

```diff
@@ -203,6 +203,7 @@ function addNavigationSpans(transaction: Transaction, entry: Record<string, any>
   addPerformanceNavigationTiming(transaction, entry, 'domContentLoadedEvent', timeOrigin);
   addPerformanceNavigationTiming(transaction, entry, 'loadEvent', timeOrigin);
   addPerformanceNavigationTiming(transaction, entry, 'connect', timeOrigin);
+  addPerformanceNavigationTiming(transaction, entry, 'secureConnect', timeOrigin, 'connectEnd');
```

Comment:
```suggestion
  addPerformanceNavigationTiming(transaction, entry, 'secureConnection', timeOrigin, 'connectEnd');
```

based on https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/secureConnectionStart